lighthouse-core/audits/byte-efficiency/uses-optimized-images.js

```diff
@@ -41,7 +40,7 @@ class UsesOptimizedImages extends Audit {
     return {
       category: 'Images',
       name: 'uses-optimized-images',
-      description: 'Has optimized images',
+      description: 'Avoids unoptimized images',
```

Comment:
>  telling users they did something when that thing doesn't apply to them 

Sort of why I think "Uses optimized images" is better than PSI's "Optimizes images". 

The first says: "All the images LH found were optimized. Good job!"
The second says: "You're (actively) optimizing images" ...which as you say, may not be the case.